cleverhans_tutorials/tutorial_models_eager.py

```diff
@@ -0,0 +1,214 @@
+"""
```

Comment:
We moved away from custom layers in tutorials, I believe the tf.layers work with tf.eager by default. You can simplify this file a lot by using them.

Reply:
above change was done after this PR is submitted, i will update this in next PR.

Reply:
I'm not sure what's best here. I'll say check with Nicolas / Ian and see what they think. I personally think it's best to have a PR with all the up-to-date code rather than pushing obsolete code at commit time.